Bottom: fa804a14603fa551f2d399664c7b12f51ed7f3bc
Top:    dd4aa5c431d99808445cf0ddf36ad11505a54816
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 01:18:43 +0000

Refresh of attempt-lesstif-gl-support

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 9d1a4a9..92d58f3 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -1586,9 +1586,9 @@ work_area_expose (Widget work_area, void *me,
   int max_x, max_y;
   Dimension width, height;
 
+  n = 0;
   stdarg (XtNwidth, &width);
   stdarg (XtNheight, &height);
-  stdarg (XmNbackground, &bgcolor);
   XtGetValues (work_area, args, n);
 
   e = &(cbs->event->xexpose);
